docs/source/release_notes.rst

```diff
@@ -13,6 +13,7 @@ Future Release
         * Temporarily restrict Dask version (:pr:`2694`)
         * Remove support for creating ``EntitySets`` from Dask or Pyspark dataframes (:pr:`2705`)
         * Bump minimum versions of ``tqdm`` and ``pip`` in requirements files (:pr:`2716`)
+        * Use ``filter`` arg in call to ``tarfile.extractall`` to safely deserialize EntitySets (:pr:`2722`)
```

Comment:
Would this be considered a breaking change?

Reply:
@jeremyliweishih I kinda debated about that - we aren't making any change to the API, so I decided against labeling it as such. It just might mean that if someone has an older environment and they upgrade FT, they might also need to upgrade Python, but only if they are using this EntitySet serialization functionality, which probably isn't all that common.

Reply:
makes sense to me!